playbook.md

```diff
@@ -178,17 +178,42 @@ Each story will also contain a list of acceptance criteria. Acceptance criteria
 
 
-#### Lifecycle of a story
-There are several states that a story has to go through in order to be deployed to production. We use tracking tools and physical story boards to keep track of which stage a given story is in.
+## Lifecycle of a story
+There are several states that a story has to go through in order to be deployed to production. We use tracking tools and physical story boards to keep track of which state a given story is in.
 
-* **Started**: A developer has started working on the story, making changes to the product to ensure that each acceptance criterion is met
-* **Code review**: The first developer has finished their work, and is ready for a second developer to [review](#Code reviews) the technical aspects of the story in a [pull request](#Pull request)
-* **Acceptance review**: The two developers have finished the review and the story is now merged, and ready for the delivery manager to review against its acceptance criteria
-* **Client review**: The delivery manager has confirmed that the story is done, and it's ready for a client to review
-* **Ready to deploy**: The client has accepted the story and it's ready to deploy
-* **Deployed**: The story is deployed to production
+### States
 
-#### Branching
+After a user need has been identified, a story is written to capture it and how we propose to meet that need with acceptance critera. Once it's written and prioritised as part of a sprint planning it will be ready to start.
+
+#### Unstarted
+- hasn't been started or isn't being actively developed yet and can be changed
+
+#### In progress
+- a developer or a pair are currently working on meeting the acceptance criteria
+
+#### Awaiting code review
+- a pull request has been opened
+- another developer is required to review the codes quality and merge into develop
+- this developer also tests the functionality with the aim of catching the majority of any bugs as early as possible and feedback them back onto the pull request
+
+#### Awaiting delivery manager review
+- the work to fulfill the story is now on staging
+- a delivery manager is required to review the product and ensure that it's ready for the client to sign off
+- a show and tell from the developer may be needed to demonstrate stories that are hard to test
+
+#### Awaiting client review
+- a delivery manager has explained what is ready and if necessary how to review it
+- the client can use Staging to see that the acceptance criteria have been met
+- based on this mark the story as accepted or rejected and include any details of which criteria hasn't been met
+
+#### Accepted
+- the client has accepted the story as the acceptance criteria on the story are met and it is ready to be deployed to production
+
+#### Deployed
+- the new story is ready to be used by the users
+- a deployment has taken place (`develop` has been merged into `master`)
```

Comment:
1. Sometimes clients want changes to be on staging but they don't want those changes to be on production. So we'll just go through and manually revert all the changes except for the change we want to push? And then when we're done will revert the commit that reverted everything and push that to the develop branch? Is that correct?
2. And sometimes when we do this, the client may be in the middle of reviewing changes - and they'll come back to us and say "well it looked okay this morning but this seems to be broken right now - what's going on?". It's going to cause delays and confusion.
3. I thought one of the complaints about git env was the fact that it's a bit tricky to rebase and some people struggled to do it properly. But reverting large amounts of changes (which may have been merged in as separate branches on different days/weeks/months) just seems extremely complex. Can you explain how you would do it?